app/views/bulletins/_form.html.erb

```diff
@@ -11,15 +11,11 @@
     </div>
   <% end %>
 
-  <div class="field">
-    <%= form.label :guild_id %>
-    <%= form.text_field :guild_id %>
-  </div>
 
-  <div class="field">
-    <%= form.label :author_id %>
-    <%= form.text_field :author_id %>
-  </div>
+  <% bulletin.author_id = current_user.id %>
+  <% bulletin.guild_id = current_user.guild_id %>
```

Comment:
If you instead `bulletin` was created by doing `current_user.bulletins.build(guild: current_user.guild)` then you wouldn't need these two lines.

Since currently the form is being rendered on the new page with this
```erb
<%= render 'form', bulletin: @bulletin %>
```
you can edit `@bulletin` in the new action of the bulletin controller
https://github.com/trevor-mill-appdev/guild-hallv1/blob/821ec61e0351171d2a42ec4a47dfdc56d59b3b37/app/controllers/bulletins_controller.rb#L16-L19

to be `@bulletin = current_user.bulletins.build(guild: current_user.guild)` instead.

Reply:
Or also you could assign these values in the `create` action too and remove them from the form partial.